### PR TITLE
build: ignore kokoro and release for synth

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -7,12 +7,16 @@
...
   '.prettierignore',
   '.prettierrc',
-  '.nycrc',
+  'codecov.yaml',
   'README.md',
-  '.github/release-please.yml',
-  '.kokoro/',
-  '.github/workflows/',
-  'codecov.yaml'
+  'renovate.json'
```
Comment (chingor13) [inline synth.py]: We don't want our standard renovate config? Or are there special configs in the templated versions that don't apply here?

File: synth.py [hunk 2]
```diff
@@ -7,12 +7,16 @@
 s.copy(templates, excludes=[
   '.eslintignore',
   '.eslintrc.yml',
+  '.github/release-please.yml',
+  '.github/workflows/',
+  '.github/publish.yml',
+  '.kokoro/*.*',
+  '.kokoro/**/*.*',
```
Comment (bcoe) [inline synth.py]: did some testing, this can just be the glob `.kokoro/**`.

File: synth.py (hunk 1 above)
Comment (JustinBeckwith) [reply synth.py]: We are supporting node.js *and* Go in this repo, so it has some customizations.  We're keeping the file, just not auto-updating.

File: synth.py (hunk 2 above)
Comment (JustinBeckwith) [reply synth.py]: Great catch!  I can't believe I missed that.

**Pull request merged**